packages/twenty-docs/docs/start/self-hosting/docker-compose.mdx

```diff
@@ -6,12 +6,14 @@ sidebar_custom_props:
 ---
 # Step by step instructions:
 
-1) Copy the [.env.example](https://github.com/twentyhq/twenty/blob/main/packages/twenty-server/.env.example) into a `.env` in the same directory where your `docker-compose.yml` file will be
-2) Run the command `openssl rand -base64 32` three times, make note of the string for each
-3) Replace the three "replace_me_with_a_random_string_access" with the three random strings you just generated.
-4) Create a `docker-compose.yml` file from the example below.
-5) Run the command `docker-compose up -d`
-6) Go to http://localhost:3001 and see your docker instance.
+1. Copy the [.env.example](https://github.com/twentyhq/twenty/blob/main/packages/twenty-server/.env.example) into a `.env` in the same directory where your `docker-compose.yml` file will be
+2. Run the command `openssl rand -base64 32` three times, make note of the string for each
+3. Replace the three "replace_me_with_a_random_string_access" with the three random strings you just generated.
```

Comment:
Could we add in the documentation : 

```
ACCESS_TOKEN_SECRET=replace_me_with_a_random_string_access
LOGIN_TOKEN_SECRET=replace_me_with_a_random_string_login
REFRESH_TOKEN_SECRET=replace_me_with_a_random_string_refresh
```

To make it more easy to look for it in the .env ?

Reply:
Could be made in another PR, I can create another issue since we're on it.